upstairs/src/upstairs.rs

```diff
@@ -945,37 +933,34 @@ impl Upstairs {
     }
 
     /// Request that the Upstairs go active
-    async fn set_active_request(
-        &mut self,
-        reply: oneshot::Sender<Result<(), CrucibleError>>,
-    ) {
-        match self.state {
+    async fn set_active_request(&mut self, res: BlockRes) {
+        match &self.state {
             UpstairsState::Initializing => {
-                self.state = UpstairsState::GoActive { reply };
+                self.state = UpstairsState::GoActive(res);
                 info!(self.log, "{} active request set", self.cfg.upstairs_id);
             }
-            UpstairsState::GoActive { .. } => {
+            UpstairsState::GoActive(..) => {
                 info!(
                     self.log,
                     "{} request to activate upstairs already going active",
                     self.cfg.upstairs_id
                 );
-                let _ = reply.send(Err(CrucibleError::UpstairsAlreadyActive));
```

Comment:
The error changed here

Reply:
Nice catch.  I must have missed that during the rebase.